Client/Frontend/Browser/BrowserViewController.swift

```diff
@@ -1696,8 +1696,11 @@ extension BrowserViewController: TabDelegate {
     }
 
     func tab(_ tab: Tab, didSelectSearchWithFirefoxForSelection selection: String) {
+        let currentChoice = NewTabAccessors.getNewTabPage(self.profile.prefs)
+        self.profile.prefs.setString(NewTabPage.blankPage.rawValue, forKey: NewTabAccessors.PrefKey)
         openBlankNewTab(focusLocationField: false, isPrivate: tab.isPrivate)
```

Comment:
The problem is probably that we are calling `openBlankNewTab` which will also load the home panels.

Reply:
If instead, We could modify `submitSearchText` and create a new `searchURLForText`. This would return a new URL and could then be used in `openURLInNewTab(URL...`